telemetrylib/SQL.cpp

```diff
@@ -11,9 +11,23 @@ class SQL : public DTI {
         this->tableToCreate = tableToCreate;
 
         restclient = new QNetworkAccessManager();
+        // Send request to create a new table when connection to server is first established
+        if(tableName.isNull() && connection) {
+            qDebug() << "Requested a new table: " << tableToCreate;
 
-        t = new std::thread(&SQL::checkConnection, this);
-        t->detach();
+            QUrl myurl;
+            myurl.setScheme("http");
+            myurl.setHost("150.136.104.125"); 
```

Comment:
Is it worth breaking this out into a config file instead of hardcoded or is it not expected to ever change?

Reply:
New member's task!

Reply:
in fact this is #48